### PR TITLE
feat: Add AUR install instructions

### PR DESCRIPTION
**Pull request by chipbuster**
Add installation instructions for starship via AUR.

#### Motivation and Context

Closes #

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,16 @@ The prompt shows information you need while you're working, while staying sleek
    $ cargo install starship
    ```
 
+   #### Arch Linux (AUR)
+
+   Starship is available on the AUR under the package name `starship`. Install it with
+
+   ```
+   # yay -S starship
```
Comment (matchai) [inline README.md]:
```suggestion
   $ yay -S starship
```

File: README.md [hunk 2]
```diff
@@ -99,6 +99,16 @@ The prompt shows information you need while you're working, while staying sleek
    $ cargo install starship
    ```
 
+   #### Arch Linux (AUR)
+
+   Starship is available on the AUR under the package name `starship`. Install it with
```
Comment (matchai) [inline README.md]: I think having this on one line may look nicer.

```suggestion
   Starship is available on the AUR under the package name `starship`. Install it with `yay` or your favorite helper.
```

File: README.md (hunk 1 above)
Comment (chipbuster) [reply README.md]: So this is a fun one.

Conventionally in Arch, the `$` sign is used for regular user prompts, while the `#` sign is used for superuser prompts.

So we can do `# yay -S starship` or `$ sudo yay -S starship` but probably not `$` without sudo.

File: README.md (hunk 2 above)
Comment (chipbuster) [reply README.md]: Agreed.

File: README.md (hunk 1 above)
Comment (matchai) [reply README.md]: Huh... Alright.
No worries 👍

File: README.md [hunk 3]
```diff
@@ -99,6 +99,14 @@ The prompt shows information you need while you're working, while staying sleek
    $ cargo install starship
    ```
 
+   #### Arch Linux (AUR)
+
+   Starship is available on the AUR under the package name `starship`. Install it with `yay` or your favorite AUR helper.
+
+   ```
+   $ sudo yay -S starship
```
Comment (iamsauravsharma) [inline README.md]: Conventionally AUR helper works without sudo they themselves asked privileged sudo access. It is believe to be bad habit of using sudo with AUR helpers.

Comment (chipbuster) [reply README.md]: You know what...I had forgotten about that.

My usual helper only escalates to sudo when needed for dependency installs, but you're right.

**Pull request merged**